notebooks/examples/join_pandas.py

```diff
@@ -0,0 +1,34 @@
+import sys
+
+import pandas as pd
```

Comment:
I'm also wondering if it's worth including a `pycytominer` merge here 🤔 (maybe not worth it, but maybe something to consider)

Reply:
Thanks I think this is a good idea! I tried to focus on the most basic examples in this PR, building a foundation of benchmarks on underlying technologies used, and would like to follow up with more complex analysis of `pycytominer` and others. I'll open an issue to explore this further.